en/micro-integrator/docs/references/connectors/importing-connector-to-integration-studio.md

```diff
@@ -0,0 +1,15 @@
+## Importing Connector to Integration Studio Solution Project
+
+1. Open WSO2 Integration Studio and create an **ESB Solution Project**.
+  <img src="../../assets/img/connectors/solution-project.png" title="Creating a new ESB Solution Project" width="800" alt="Creating a new ESB Solution Project" />
+
+2. Right click on the project that you created and click on **Add or Remove Connector** -> **Add Connector**.
+<img src="../../assets/img/connectors/add-connector.png" title="Add or Remove Connector" width="800" alt="Add or Remove Connector" />
+
+3. You are directed to the WSO2 Connector Store. 
```

Comment:
This is not a step. Merge this with the previous numbered item.

Reply:
Done